CK3toEU4/Data_Files/blankMod/output/common/disasters/decline_of_mali.txt

```diff
@@ -1 +1,37 @@
-##Makes no sense to have in an alternate history scenario
\ No newline at end of file
+##Makes no sense to have in an alternate history scenario
+decline_of_mali = {
```

Comment:
Are you sure we want this in here?

Reply:
Yes, previously it was just blanked out, but it made the error log unhappy, so I just made sure it would never fire.